racktables/racktables.py

```diff
@@ -18,6 +18,11 @@ def search(self, search_payload={}):
             join(self.url, "index.php"),
             params="&".join("%s=%s" % (k, v) for k, v in search_payload.items()),
         )
+        status = req.status_code
+        if status == 401:
+          raise Exception("Racktables returned 401 Unauthorized. Are your credentials correct?")
+        elif status >= 300:
+          raise Exception(f"Racktables returned statuscode {status} for a search request. Manual investigation needed.")
```

Comment:
Might be worth including the search terms if we can? To make it easy to reproduce

Reply:
I tried to incorporate your feedback. Are you fine with that?